README.md

```diff
@@ -167,6 +168,27 @@ expect(descendant).not.toContainElement(ancestor)
 // ...
 ```
 
+### `toContainHtml`
+
+```typescript
+toContainHtml(htmlText: String)
+```
+
+Assert whether a string representing a HTML element is contained in another Element:
+
+```javascript
+// add the custom expect matchers once
+import 'jest-dom/extend-expect'
+
+// ...
+// <span data-testid="parent"><span data-testid="child"></span></span>
+const parent = queryByTestId('parent')
+expect(parentElement).toContainHtml('<span data-testid="child"></span>')
+// ...
+```
+
+> Note: Chances are you probably do not need to use this method. We encourage testing from the perspective of how the user perceives the app in a browser. That's why testing against a specific DOM structure is not advised. It might come in handy in rare cases such as if your users manipulate the DOM such as a content management tool - where users are expected to type custom HTML. please use: [`toContainElement`](#tocontainelement)
```

Comment:
You could probably leave out "It might come in handy in rare cases such as if your users manipulate the DOM such as a content management tool - where users are expected to type custom HTML".

And capital `P` on "please".